### ISSUE TITLE
[Adaptive Card iOS - Input.Text.Style.Json]: Voice over is conveying the irrelevant information as double tap to edit for the non interactive 'Specify the type of text being… heading' text.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8430]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.Text.Style.Json' and activate it.
7. Now, perform the right swipe and navigate to the 'specify the type of text being requested' heading and observe the issue. 

**Actual Result:**
While navigating to the 'specify the type of text being requested' heading, voice over is conveying the irrelevant information as double tap to edit whereas it is non editable field.

**Expected Result:**
Voice over should not convey the irrelevant information for the heading. It should announce only the relevant information for the heading.

**User Impact:**
It will impact the user who depends on the screen reader as they might be confused about the field and treat as editable field for the non editable fields.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/228814970-bd2b46ab-0091-468d-96bc-c78ddb625a28.mov

### Card JSON

```JSON
NA
```
